backend/src/test/scala/cromwell/backend/RuntimeAttributeValidationSpec.scala

```diff
@@ -0,0 +1,11 @@
+package cromwell.backend
+
+import org.scalacheck.Properties
+import org.scalacheck.Prop._
+
+class RuntimeAttributeValidationSpec extends Properties("Runtime Validation") {
+
+  property("foo") = secure{ true}
```

Comment:
um

Reply:
yeah this is WIP , I'll prefix my commits with such in the future